pkg/cluster/ipaddresses.go

```diff
@@ -130,11 +130,13 @@ func (m *manager) populateDatabaseIntIP(ctx context.Context) error {
 	if m.doc.OpenShiftCluster.Properties.APIServerProfile.IntIP != "" {
 		return nil
 	}
+	infraID := m.doc.OpenShiftCluster.Properties.InfraID
+	if infraID == "" {
+		infraID = "aro"
+	}
```

Comment:
Why are we doing this in just one place?  Do we not have a defaulting rule we should set in all places we are getting InfraID?  May this a utility function and call it with `OpenShiftCluster` as the arg?

Reply:
We do this in all places where we need to do this (not all places needed). And after "fixup" runs this can be removed. So I dont want to populate code with more change :/